src/components/ContactPolicy/ContactPolicy.test.js

```diff
@@ -131,6 +171,50 @@ describe('ContactPolicy component', () => {
     inputInterval = await screen.findByRole('spinbutton', { name: 'contact_policy.interval_unit' });
     expect(inputInterval).toHaveValue(14);
 
+    // Enable "Time Slot" switch button
+    let switchButtonTimeSlot = screen.getByRole('checkbox', {
+      name: 'contact_policy.time_restriction.time_slot_toggle_text',
+    });
+    expect(switchButtonTimeSlot).not.toBeChecked();
+    await act(() => user.click(switchButtonTimeSlot));
+    switchButtonTimeSlot = await screen.findByRole('checkbox', {
+      name: 'contact_policy.time_restriction.time_slot_toggle_text',
+    });
+    expect(switchButtonTimeSlot).toBeChecked();
+
+    // Set "Hour From" = 10
+    let inputHourFrom = screen.getByRole('spinbutton', {
+      name: 'contact_policy.time_restriction.hour_from_aria_label',
+    });
+    await act(() => user.clear(inputHourFrom));
+    await act(() => user.type(inputHourFrom, '10'));
+    inputHourFrom = await screen.findByRole('spinbutton', {
+      name: 'contact_policy.time_restriction.hour_from_aria_label',
+    });
+    expect(inputHourFrom).toHaveValue(10);
+
+    // Set "Hour To" = 20
+    let inputHourTo = screen.getByRole('spinbutton', {
+      name: 'contact_policy.time_restriction.hour_to_aria_label',
+    });
+    await act(() => user.clear(inputHourTo));
+    await act(() => user.type(inputHourTo, '20'));
```

Comment:
Is the act's result _awaitable_?

Reply:
Mm, I am not sure, but this is being treated in this way in all tests.